localstack/utils/bootstrap.py

```diff
@@ -360,6 +361,56 @@ def start_infra_locally():
     return infra.start_infra()
 
 
+def validate_localstack_config(name):
+    dirname = os.getcwd()
+    compose_file_name = '%s/%s' % (dirname, name)
+
+    # validating docker-compose file
+    cmd = "docker-compose -f '%s' config" % (compose_file_name)
+    try:
+        run(cmd)
+    except Exception:
+        return 'docker-compose file is not valid'
+
+    # validating docker-compose variable
+    with open(compose_file_name) as file:
+        compose_content = yaml.full_load(file)
+        localstack_service = [service for service in compose_content['services']
+            if compose_content['services'][service]['image'] in constants.OFFICIAL_IMAGES]
+        if len(localstack_service) > 0:
+            localstack_service = localstack_service[0]
+        else:
+            raise Exception('No official docker image found. Please use one of this image: %s'
+                % (constants.OFFICIAL_IMAGES))
+
+    network_mode = compose_content['services'][localstack_service]['network_mode']
+    image_name = compose_content['services'][localstack_service]['image']
+    container_name = compose_content['services'][localstack_service].get('container_name')
+    docker_ports = (port.split(':')[0] for port in compose_content['services'][localstack_service]['ports'])
+    docker_env = dict((env.split('=')[0], env.split('=')[1])
+        for env in compose_content['services'][localstack_service]['environment'])
+
+    # docker-compose file validation cases
+    if (docker_env.get('LAMBDA_REMOTE_DOCKER') == constants.FALSE_STRINGS and
```

Comment:
Should be
```
docker_env.get('LAMBDA_REMOTE_DOCKER') in constants.FALSE_STRINGS
```
instead of
```
docker_env.get('LAMBDA_REMOTE_DOCKER') == constants.FALSE_STRINGS
```